### DISCUSSION TITLE
已经开了一条http的隧道，如何将其转成https？

### DISCUSSION THREAD

**Discussion by JZLeung** [natfrp/wiki#145] [使用问题]
有一条隧道A，绑定域名 `api.example.com`，代理本地的 `http://127.0.0.1:3000` 的地址，但是并没有 `https`。

在腾讯云免费申请了一个 SSL 证书，要怎么操作才能让 `api.example.com` 以 https 的形式访问？

**Comment by Klosw** [comment]
你的服务器得先支持访问 https 吧

**Comment by MaXin-noob** [comment]
保证你的服务器可以通过HTTPS访问，可以通过内网直接访问ip地址，它提示证书错误（此证书来自XXXXX，但是目前网站为XXXX，意思是你的证书和网站域名不匹配）说明你配置成功了，然后直接开一条HTTPS隧道映射到443端口就可以
